### ISSUE TITLE
glib stuck while upgrading

### ISSUE THREAD

**Issue by Frizlab** [Homebrew/homebrew#37515]
I'm running `brew upgrade --build-from-source` and the upgrade is stuck while running `ed` while upgrading `glib`.
Here's the full output of the upgrade process: https://gist.github.com/Frizlab/152750c813f98e6010d0
I've added a marker in the output to show where the process is stuck (lines 716 to 718). The output after the marker is generated after I unstuck the process by running (in another Terminal): `killall ed`.

**Comment by timiyay** [comment]
I'm also getting the same issue. Upgrading `glib` (no flags) hangs at the following line:
```bash
ed -s - config.h <config.h.ed
```
I left it going last night, and it was still hung this morning. Furthermore, it had filled my entire disk, which required a system reset to clear.

I'm assuming it relates to the recent formula update for `glib` in 66c012.

**Comment by jacknagel** [comment]
Should be fixed in 3bc26d890abb380644bf5e60565f19a039ab8811.

@Noctem @mikemcquaid these are audit false positives:

```
$ brew audit glib --strict
==> brew style glib

1 file inspected, no offenses detected

==> audit problems
glib:
 * Use `system "ed", "-s", "-", "config.h", "<config.h.ed"` instead of `system "ed -s - config.h <config.h.ed"`
 * Use `system "ulimit", "-n", "1024;", "make", "check"` instead of `system "ulimit -n 1024; make check"`

Error: 2 problems in 1 formulae
```

Changes of this nature need to be more carefully reviewed and tested in the future. Thanks!

**Comment by mikemcquaid** [comment]
@jacknagel My bad, thanks and sorry.
